examples/postgresql-ephemeral-template.json

```diff
@@ -127,11 +127,11 @@
                   }
                 },
                 "livenessProbe": {
+                  "exec": {
+                    "command": [ "/bin/sh", "-i", "-c", "pg_isready -h 127.0.0.1 -p 5432" ]
```

Comment:
I guess the `/bin/sh` is necessary, but I would rather avoid using the `-i` option, if possible.

Reply:
I have left it in only because it was present in other /bin/sh invocations inside the template